easy_my_coop/models/coop.py

```diff
@@ -437,14 +449,16 @@ def get_partner_company_vals(self):
                         'last_name': self.company_name,
                         'is_company': self.is_company,
                         'company_register_number': self.company_register_number, #noqa
-                        'customer': False, 'cooperator': True,
+                        'cooperator': True,

```

Comment:
The customer key was removed, because it was duplicated. See line 456 : https://github.com/coopiteasy/vertical-cooperative/pull/2/files#diff-e00ff57c58fa1904790e8f5bcc5e5881L444